commands/create/cluster/command.go

```diff
@@ -51,23 +52,30 @@ type Arguments struct {
 	Owner                 string
 	ReleaseVersion        string
 	Scheme                string
+	MasterHA              *bool
 	UserProvidedToken     string
 	Verbose               bool
 }
 
 // collectArguments gets arguments from flags and returns an Arguments object.
-func collectArguments() Arguments {
+func collectArguments(cmd *cobra.Command) Arguments {
 	endpoint := config.Config.ChooseEndpoint(flags.APIEndpoint)
 	token := config.Config.ChooseToken(endpoint, flags.Token)
 	scheme := config.Config.ChooseScheme(endpoint, flags.Token)
 
+	var haMasters *bool
```

Comment:
If the flag has been set by the user, then we'll actually set the value... Otherwise, leave it nil